build/Makefile.in

```diff
@@ -583,6 +587,10 @@ switch no-tracing no-cgoto:
 	$(MSG) generating assembler for $@
 	$(CMD)$(CC) @asmswitch@ $(CFLAGS) @objflags@ $(CINCLUDES) @asmout@$@ $*.c
 
+@asm@@obj@:
+	$(MSG) Assembling $@
+	$(CMD)$(AS) $(ASFLAGS) @ccswitch@ @objout@$@ $*@asm@
```

Comment:
I don't see the assembly this was added in aid of; is it being retained so we don't have to re-figure it out in the future?

Reply:
Yes :smile:  it worked and I didn't want to have to figure it out again in case we ever wanted to do something with assembly again.